src/app/legacy/config.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  BasicIssuanceModuleAddress,
+  IndexDebtIssuanceModuleV2Address,
+  IndexDebtIssuanceModuleV2Address_v2,
+} from '@indexcoop/flash-mint-sdk'
+
+import {
+  BedIndex,
+  GitcoinStakedETHIndex,
+  LeveragedRethStakingYield,
+} from '@/constants/tokens'
+
+export const Issuance = {
+  [BedIndex.symbol]: BasicIssuanceModuleAddress,
+  [GitcoinStakedETHIndex.symbol]: IndexDebtIssuanceModuleV2Address,
+  [LeveragedRethStakingYield.symbol]: IndexDebtIssuanceModuleV2Address_v2,
+}
```

Comment:
Not using this [function](https://github.com/IndexCoop/flash-mint-sdk/blob/main/src/utils/issuanceModules.ts#L30) from the FlashMintSDK as it might not always support all legacy tokens but good reference to cross check some of the used issuance addresses here.